specs/altair/fork.md

```diff
@@ -84,6 +99,8 @@ def upgrade_to_altair(pre: phase0.BeaconState) -> BeaconState:
         # Inactivity
         inactivity_scores=[uint64(0) for _ in range(len(pre.validators))],
     )
+    # Fill in previous epoch participation from the pre state's pending attestations
+    translate_participation(post, pre.previous_epoch_attestations)
```

Comment:
just echoing the logic here: the last epoch transition of phase 0 will have already incorporated the attestations from the penultimate epoch so then we are interested in translating the attestations from the last epoch of phase 0.

by the time this code executes, those specific attestations will be residing in `pre.previous_epoch_attestations`

Reply:
yse